jaxrs/src/test/java/com/cerner/beadledom/jaxrs/provider/FakeModel.java

```diff
@@ -28,6 +28,10 @@
   @JsonProperty("inner_models")
   public List<FakeInnerModel> innerModels;
 
+  public FakeModel() {
```

Comment:
We didn't break of the existing tests using this model did we? I can't imagine so but you never know.